File: src/main/java/ru/ioleksiv/telegram/bot/api/model/objects/inline/queryresult/InlineQueryResultCachedSticker.java
package ru.ioleksiv.telegram.bot.api.model.objects.inline.queryresult;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * @see <a href="https://core.telegram.org/bots/api#inlinequeryresultcachedsticker>InlineQueryResultCachedSticker</a>
 */
public class InlineQueryResultCachedSticker extends ContentInlineResult {
    private static final String TYPE = "sticker";
    /**
     * sticker_file_id	String	A valid file identifier of the sticker
     */
    @JsonProperty("sticker_file_id")
    private String stickerFileId = null;

    InlineQueryResultCachedSticker() {
        super(TYPE);
    }

    public String getStickerFileId() {
        return stickerFileId;
    }

    public void setStickerFileId(String stickerFileId) {
        this.stickerFileId = stickerFileId;
    }

}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/InlineQueryResultCachedSticker.kt
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * @see [InlineQueryResultCachedSticker](https://core.telegram.org/bots/api/#inlinequeryresultcachedsticker)
 */
class InlineQueryResultCachedSticker(
        /**
         * sticker_file_id String A valid file identifier of the sticker
         */
        @JsonProperty("sticker_file_id")
        var stickerFileId: String,
        /**
         * type String Type of the result
         */
        @JsonProperty("type")
        val type: String = "sticker"
) : ContentInlineResult()
